src/medali/lib/tempinator/V1M0.ini

```diff
@@ -44,26 +44,20 @@ run_number: integer
 # data set creator
 creator: string
 
-# band name (e.g., polarisation or optical band)
-band: string
-
 # name of the variable to be aggregated (e.g., sig0, gam0, plia, ...)
 variable_name: string
 
-# version of the input data used for temporal aggregation (e.g. V01R01)
-input_data_version: string
-
-# acquisition mode id
-mode: string
+# verbal description how the values were encoded
+encoding_info: string
 
-# scale factor
-scale_factor: integer
+# verbal description of the values
+value_info: string
 
-# if set, the aggregation is computed per extra field, e.g. D090 (per relative orbit)
-per_extra_field: boolean
```

Comment:
Where is this information going to now?

Reply:
Nowhere actually, since the new version of `tempinator` has a more generic way of filtering and does not use such flags anymore. Since its more generic, I did not want to put this into the metadata, since the filename anyway reflects how one aggregates the data.